test/runtests.jl

```diff
@@ -35,6 +35,17 @@ end
     @test checkbounds(Bool, sv, 123, 123)
 end
 
+@testset "bringwithin" begin
```

Comment:
Maybe add a test for axes with offsets, like `ShiftedArrays.bringwithin(11, 2:7)`